source/more-info/unsupported/container.markdown

```diff
@@ -0,0 +1,14 @@
+---
+title: "Containers known to cause issues"
+description: "More information on why certain containers marks the installation as unsupported."
+---
+
+## The issue
+
+The supervisor needs to be the only manager of the containers that it manages, unfortunately there are som other containers that also take on this role, and even worse they will auto-update the containers to development builds without consideration of the channel you have chosen, because of this a few containers known to cause issues have been added to a denylist that will flag your installation as unsupported.
+
+## The solution
+
+You will find a list of offending containers in your supervisor log.
+
+If you remove all offending containers from your host and then reload the supervisor it will no longer be marked as unsupported for this reason.
```

Comment:
```suggestion
If you remove all offending containers from your host and then reload the Supervisor it will no longer be marked as unsupported for this reason.
```